k8s-artifacts/k8s_deployments.yaml

```diff
@@ -0,0 +1,103 @@
+# Deployments for Adapter, router and enforcer
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: adapter
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: adapter
+  template:
+    metadata:
+      labels:
+        app: adapter
+    spec:
+      containers:
+        - name: adapter
+          volumeMounts:
+            - mountPath: /home/wso2/security
+              name: adapter-security-vol
+            - mountPath: /home/wso2/conf
+              name: adapter-conf-vol
+          image: wso2/mg-adapter:4.0.0-m1
+          ports:
+            - containerPort: 18000
+            - containerPort: 9843
+      volumes:
+        - name: adapter-security-vol
+          configMap:
+            name: adapter-security
+        - name: adapter-conf-vol
+          configMap:
+            name: conf
+      restartPolicy: Always
```

Comment:
We need to add liveness, readiness probes here. Applies to all the deployments.

Reply:
Fixed in ffe857e265a212eddc9148eae18bfe0e341c3148